src/Compiler/Driver/ReuseTcResults/ReuseTcResultsDriver.fs

```diff
@@ -0,0 +1,124 @@
+module internal FSharp.Compiler.ReuseTcResults
+
+open System
+open System.Collections.Generic
+open System.IO
+
+open FSharp.Compiler.CompilerConfig
+open FSharp.Compiler.Diagnostics
+open FSharp.Compiler.GraphChecking
+open FSharp.Compiler.IO
+open FSharp.Compiler.Syntax
+open FSharp.Compiler.Syntax.PrettyNaming
+
+open Internal.Utilities.Hashing
+
+let (++) a b = Path.Combine(a, b)
+
+[<Sealed>]
+type ReuseTcResultsDriver(tcConfig: TcConfig) =
```

Comment:
Should probably be named differently, something along the lines of `CachingDriver`.

Reply:
Alright, renamed